cli/node/memcoin/mod_test.go

```diff
@@ -52,7 +53,7 @@ func TestMemcoin_Scenario_SetupAndTransactions(t *testing.T) {
 		wg.Wait()
 	}()
 
-	require.True(t, waitDaemon(t, []string{node1, node2, node3}), "daemon failed to start")
+	require.True(t, waitDaemon([]string{node1, node2, node3}), "daemon failed to start")
```

Comment:
in a future PR, we might want to change the API such that the call looks like
```suggestion
	requireDaemonStart(t, node1, node2, node3)
```
given `t` and `"daemon failed to start"` are the same in all tests.